driver/parallel.go

```diff
@@ -160,12 +158,10 @@ func (pg *parallelGroup) nextSourceForConn(conn *client.Connection) (ScannerClos
 			zbuf.Scanner
 			io.Closer
 		}{s, rc}
-
 		pg.mu.Lock()
 		pg.scanners[sc] = struct{}{}
 		pg.mu.Unlock()
 		return sc, nil
-
```

Comment:
I would just assume leaving these lines as is considering you're not really doing work over here and line spacing is a matter of preference.

Reply:
I put the spaces in there to begin with, and I regretted it, because it was inconsistent with the style of the rest of the code in this source file.

Reply:
I'm with @mattnibs on this one. The substantive changes here are already pretty large, and adding cosmetic changes on top that increases the burden on reviewers. I always appreciate a smaller diff.